File: swift/iniciante/1012.swift
import Foundation

let input = readLine()!.split(separator: " ").map { String($0) }
let A = Double(input[0].replacingOccurrences(of: ",", with: "."))!
let B = Double(input[1].replacingOccurrences(of: ",", with: "."))!
let C = Double(input[2].replacingOccurrences(of: ",", with: "."))!

let pi = 3.14159

let triangleArea = (A * C) / 2.0
let circleArea = pi * pow(C, 2)
let trapezoidArea = ((A + B) * C) / 2.0
let squareArea = pow(B, 2)
let rectangleArea = A * B

print(String(format: "TRIANGULO: %.3f", triangleArea))
print(String(format: "CIRCULO: %.3f", circleArea))
print(String(format: "TRAPEZIO: %.3f", trapezoidArea))
print(String(format: "QUADRADO: %.3f", squareArea))
print(String(format: "RETANGULO: %.3f", rectangleArea))

File: javascript/iniciante/1012.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
});

const values = [];

rl.on("line", (line) => {
  values.push(...line.trim().split(/\s+/).filter(Boolean));

  if (values.length >= 3) {
    const A = Number(values[0]);
    const B = Number(values[1]);
    const C = Number(values[2]);
    const pi = 3.14159;
    const triangleArea = (A * C) / 2.0;
    const circleArea = pi * Math.pow(C, 2);
    const trapezoidArea = ((A + B) * C) / 2.0;
    const squareArea = Math.pow(B, 2);
    const rectangleArea = A * B;
    console.log(`TRIANGULO: ${triangleArea.toFixed(3)}`);
    console.log(`CIRCULO: ${circleArea.toFixed(3)}`);
    console.log(`TRAPEZIO: ${trapezoidArea.toFixed(3)}`);
    console.log(`QUADRADO: ${squareArea.toFixed(3)}`);
    console.log(`RETANGULO: ${rectangleArea.toFixed(3)}`);
    rl.close();
  }
});
